index.html

```diff
@@ -209,6 +209,16 @@ <h3 id="server-performance"><a href="#server-performance">Server performance</a>
             <td class="alert">no</td>
           </tr>
           <tr>
+            <td><a href="https://iojs.org/api/https.html">io.js</a></td>
+            <td class="ok">yes</td>
+            <td class="ok"><a href="https://github.com/joyent/node/pull/5882">yes</a></td>
```

Comment:
Oh, I see. So, io.js will automatically issue both a ticket and session-ID? Is there a way to toggle this stuff? ... I'd love to see some docs on this stuff ;-)